user-key-value-api-spec.yaml

```diff
@@ -0,0 +1,193 @@
+openapi: 3.0.0
```

Comment:
Let's use version 3.0.3, which is NOT the latest OAS, but SmartAPI doesn't support 3.0.4 or the newer 3.1.x yet.

Reply:
changed